shared/fs/folder/offline.js

```diff
@@ -4,17 +4,28 @@ import React from 'react'
 import * as Styles from '../../styles/index'
 import * as Types from '../../constants/types/fs'
 import TopBar from '../top-bar'
+import * as Constants from '../../constants/fs'
+import {namedConnect} from '../../util/typed-connect'
 
 type Props = {|
   path: Types.Path,
+  syncEnabled: boolean,
 |}
 
 const OfflineFolder = (props: Props) => (
   <Kb.Box2 direction="vertical" style={styles.contentContainer} fullWidth={true} alignItems={'stretch'}>
     <TopBar path={props.path} />
     <Kb.Box2 direction="vertical" style={styles.emptyContainer} fullWidth={true} centerChildren={true}>
-      <Kb.Icon type="iconfont-cloud" sizeType={'Huge'} color={Styles.globalColors.black_10} />
-      <Kb.Text type="BodySmall">You haven't synced this folder.</Kb.Text>
+      <Kb.Icon
+        type={props.syncEnabled ? 'iconfont-time' : 'iconfont-cloud'}
+        sizeType={'Huge'}
+        color={Styles.globalColors.black_10}
+      />
+      <Kb.Text type="BodySmall">
+        {props.syncEnabled
+          ? 'This folder will sync once you get back online.'
+          : "You haven't synced this folder."}
```

Comment:
Why single quotes above but double quotes here?

Reply:
Huh, I wrote it with both having single quotes, `prettier` must have done this. I guess it's because the second one has a `'` in it.